include/json/value.h

```diff
@@ -1082,6 +1082,10 @@ class JSON_API ValueIterator : public ValueIteratorBase {
 
 } // namespace Json
 
+/// Specialize std::swap() for Json::Value.
+namespace std { void swap(Json::Value&, Json::Value&); }
```

Comment:
This is an illegal declaration according to [namespace.std], because it's declaring a new function in namespace std.

The standard way to define a swap is in the namespace of the type, in this case namespace Json. Then it will be found by ADL, as expected by the Swappable concept.